tests/integration/apigateway/test_apigateway_dynamodb.snapshot.json

```diff
@@ -77,7 +85,7 @@
           "date": "date",
           "x-amz-apigw-id": "x-amz-apigw-id",
           "x-amzn-requestid": "<uuid:1>",
-          "x-amzn-trace-id": "Root=1-640e7ae9-63faf1aa71313dd3638b68fd"
+          "x-amzn-trace-id": "Root=1-64afea44-3087b2f50c440c1958c4d62e"
```

Comment:
This is actually an invalid snapshot (before and now) since it won't validate against AWS. The unique value here would need to be replaced via a transformer to make it valid. 

Since this hasn't been introduced in this PR, feel free to leave as-is though.